--- ./configure.in.org	Tue Jan 28 19:42:44 2003
+++ ./configure.in	Tue Jan 28 19:43:52 2003
@@ -188,7 +188,6 @@
 packagesrcdir=`cd $srcdir && pwd`
 AC_DEFINE_UNQUOTED(PACKAGE_SOURCE_DIR, "${packagesrcdir}/gcvs")
 
-AC_CONFIG_SUBDIRS(cvsunix)
 AC_OUTPUT([
 	Makefile \
 	common/Makefile \
--- ./Makefile.am.org	Tue Jan 28 19:42:43 2003
+++ ./Makefile.am	Tue Jan 28 19:44:23 2003
@@ -1,5 +1,5 @@
 EXTRA_DIST = GCVS-VERSION .cvsignore PORTING README.SUN gcvs.spec make_configure
-SUBDIRS = common cvstree rf gcvs Macros cvsunix GuiDoc
+SUBDIRS = common cvstree rf gcvs Macros GuiDoc
 CONFIG_STATUS_DEPENDENCIES = GCVS-VERSION
 
 etags:
