diff -Naur gcvs-1.0a6.orig/Makefile.am gcvs-1.0a6/Makefile.am
--- gcvs-1.0a6.orig/Makefile.am	Wed Jul  5 07:17:42 2000
+++ gcvs-1.0a6/Makefile.am	Mon Jul 15 11:23:03 2002
@@ -1,5 +1,5 @@
 EXTRA_DIST = GCVS-VERSION .cvsignore PORTING README.SUN gcvs.spec
-SUBDIRS = common cvstree rf gcvs Macros cvsunix GuiDoc
+SUBDIRS = common cvstree rf gcvs Macros GuiDoc
 
 etags:
 	etags -C `find . -name \*.cpp`
diff -Naur gcvs-1.0a6.orig/configure.in gcvs-1.0a6/configure.in
--- gcvs-1.0a6.orig/configure.in	Fri Aug 18 06:57:25 2000
+++ gcvs-1.0a6/configure.in	Mon Jul 15 11:25:16 2002
@@ -156,7 +156,6 @@
 packagesrcdir=`cd $srcdir && pwd`
 AC_DEFINE_UNQUOTED(PACKAGE_SOURCE_DIR, "${packagesrcdir}/gcvs")
 
-AC_CONFIG_SUBDIRS(cvsunix)
 AC_OUTPUT(
 	Makefile \
 	common/Makefile \
--- gcvs-1.0a6/common/AppGlue.cpp~	Mon Jul 15 13:31:59 2002
+++ gcvs-1.0a6/common/AppGlue.cpp	Mon Jul 15 13:31:59 2002
@@ -879,16 +879,6 @@
 		cvscmd << kPathDelimiter;
 	cvscmd << "cvs";
 	UStr firstCmd = cvscmd;
-	if(stat(cvscmd, &sb) == -1 || !(S_ISREG(sb.st_mode) || S_ISLNK(sb.st_mode)))
-	{
-		cvscmd = PACKAGE_SOURCE_DIR;
-		CStr uppath, folder;
-		SplitPath(cvscmd, uppath, folder);
-		cvscmd = uppath;
-		if(!cvscmd.endsWith(kPathDelimiter))
-			cvscmd << kPathDelimiter;
-		cvscmd << "cvsunix/src/cvs";
-	}
 
 	if(stat(cvscmd, &sb) == -1 || !(S_ISREG(sb.st_mode) || S_ISLNK(sb.st_mode)))
 	{
